packages/vsx-registry/src/browser/vsx-extensions-contribution.ts

```diff
@@ -75,8 +96,36 @@ export class VSXExtensionsContribution extends AbstractViewContribution<VSXExten
             priority: 1,
             onDidChange: this.model.onDidChange
         });
+
+        this.registerMoreToolbarItem({
+            id: VSXExtensionsCommands.INSTALL_FROM_VSIX.id,
+            command: VSXExtensionsCommands.INSTALL_FROM_VSIX.id,
+            tooltip: VSXExtensionsCommands.INSTALL_FROM_VSIX.label,
+            group: 'other_1'
+        });
     }
 
+    /**
+     * Register commands to the `More Actions...` extensions toolbar item.
+     */
+    public registerMoreToolbarItem = (item: Mutable<TabBarToolbarItem>) => {
```

Comment:
Please remove `public`. This project tries to avoid using `public`. It's the default visibility. Thanks!